src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -428,23 +428,29 @@ public void UpdateGlucoseReduction(float value)
     {
         var percentage = value * 100 + "%";
 
-        glucoseReductionLabel.Text = "The amount of glucose has been reduced to " + percentage +
-            " of the previous amount.";
+        // The amount of glucose has been reduced to {0} of the previous amount.
+        glucoseReductionLabel.Text =
+            string.Format(CultureInfo.CurrentCulture,
+                TranslationServer.Translate("MICROBE_EDITOR_GLUCOSE_RED"),
```

Comment:
ID string shouldn't be abbreviated.
This is as far as I got reviewing...
There's probably similar problems all around which should be addressed everywhere before I'll take a look at the earlier parts to see if they are resolved and finish looking at the rest of the changes.